tests/main.nf.test

```diff
@@ -8,7 +8,7 @@ nextflow_pipeline {
 
         when {
```

Comment:
Could you add test cases for the following:

1. The `sample_name` contains spaces/special characters (make sure they are replaced properly).
2. Missing `sample_name` and only `sample` (make sure `meta.id` is set properly).
3. Two identical `sample_name` values (make sure they are made unique in the pipeline)

Reply:
Echoing test case 3.